packages/modal/src/Modal.tsx

```diff
@@ -48,16 +48,21 @@ const visibleBackdrop = css`
 
```

Comment:
While updating this file, would you mind switching the top-right "X" button to an `IconButton`?